packages/plugins/execute-subscription-event/src/index.ts

```diff
@@ -18,7 +18,7 @@ export type ContextFactoryType<TContextValue = DefaultContext> = (
   options: ContextFactoryOptions
 ) => PromiseOrValue<ContextFactoryHook<TContextValue> | void>;
 
-export const useExtendContextValuePerExecuteSubscriptionEvent = <TContextValue = unknown>(
+export const useExtendContextValuePerExecuteSubscriptionEvent = <TContextValue extends Record<any, any>>(
```

Comment:
technically this is a breaking change 🤔

Reply:
but could also be considered as a bug fix, as the earlier type did not satisfy the usage constraint...